docs/src/main/paradox/external/apache-camel.md

```diff
@@ -0,0 +1,11 @@
+# Apache Camel
+
+**Apache Camel endpoints** can be integrated into Akka Stream applications with the Streamz library.
+
+Learn more about it at
+
+- [GitHub krasserm/streamz](https://github.com/krasserm/streamz)
+- [Camel Scala DSL for Akka Streams](https://github.com/krasserm/streamz/blob/master/streamz-camel-akka/README.md#scala-dsl)
+- [Camel Java DSL for Akka Streams](https://github.com/krasserm/streamz/blob/master/streamz-camel-akka/README.md#java-dsl)
+
+This library is not maintained in the Alpakka repository.
```

Comment:
Maybe here and in other external connector pages move this sentence before "Learn more..."? So it would read: